scss/application.scss

```diff
@@ -0,0 +1,156 @@
+
+
```

Comment:
No need for this 2 spaces :stuck_out_tongue_closed_eyes:

Reply:
And remember this `application.scss` should have only the imports of the files from your site :smile: